hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/validator/ContentTypeValidator.java

```diff
@@ -0,0 +1,26 @@
+package pl.allegro.tech.hermes.management.domain.topic.validator;
+
+import org.springframework.stereotype.Component;
+import pl.allegro.tech.hermes.api.ContentType;
+import pl.allegro.tech.hermes.management.config.TopicProperties;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+@Component
+public class ContentTypeValidator {
+
+    private static final String ERROR_MESSAGE = "Content type %s is not within allowed content types [%s]";
```

Comment:
Nice to have:
I tested this feature and I got error message: `{"message":"Content type JSON is not within allowed content types [[AVRO]]","code":"VALIDATION_ERROR"}`. Allowed content types could be printed as `%s` instead of `[%s]`.

Reply:
Done.